src/Bigtable/Connection/Grpc.php

```diff
@@ -0,0 +1,159 @@
+
+    /**
+     * @var tableAdminClient
+     */
+    private $tableAdminClient;
+
+    /**
+     * @var array
+     */
+    private $lroResponseMappers = [
+        [
+            'method' => 'createInstance',
+            'typeUrl' => 'type.googleapis.com/google.bigtable.admin.instance.v2.CreateInstanceMetadata',
+            'message' => Instance::class
+        ], [
+            'method' => 'updateInstance',
+            'typeUrl' => 'type.googleapis.com/google.bigtable.admin.instance.v2.UpdateInstanceMetadata',
+            'message' => Instance::class
+        ]
+    ];
+    
+    /**
+     * @param array $config [optional]
+     */
+    public function __construct(array $config = [])
+    {
+        $this->serializer = new Serializer([
+            'commit_timestamp' => function ($v) {
+                return $this->formatTimestampFromApi($v);
+            },
+            'read_timestamp' => function ($v) {
+                return $this->formatTimestampFromApi($v);
+            }
+        ], [
+            'google.protobuf.Value' => function ($v) {
+                return $this->flattenValue($v);
+            },
+            'google.protobuf.ListValue' => function ($v) {
+                return $this->flattenListValue($v);
+            },
+            'google.protobuf.Struct' => function ($v) {
+                return $this->flattenStruct($v);
+            },
+        ]);
+
+        $config['serializer'] = $this->serializer;
+        $this->setRequestWrapper(new GrpcRequestWrapper($config));
+
+        $this->instanceAdminClient = new BigtableInstanceAdminClient();
+        $this->bigtableClient = new BigtableGapicClient();
+        $this->tableAdminClient = new BigtableTableAdminGapicClient();
+    }
+
+    /**
+     * @param array $args
+     */
+    public function listInstances(array $args)
+    {
+        $projectId = $this->pluck('projectId', $args);
+        return $this->send([$this->instanceAdminClient, 'listInstances'], [
+            $projectId,
+            $this->addResourcePrefixHeader($args, $projectId)
+        ]);
+    }
+
+    /**
+     * @param array $args
+     */
+    public function listTables(array $args)
+    {
+        $parent = $this->pluck('name', $args);
+        return $this->send([$this->tableAdminClient, 'listTables'], [
+            $parent,
+            $this->addResourcePrefixHeader($args, $parent)
+        ]);
+    }
+
+    /**
+     * @param array $args
+     */
+    public function deleteTable(array $args)
+    {
+        $tableName = $this->pluck('name', $args);
+        return $this->send([$this->tableAdminClient, 'deleteTable'], [
+            $tableName,
+            $this->addResourcePrefixHeader($args, $tableName)
+        ]);
+    }
+
+    /**
+     * Add the `google-cloud-resource-prefix` header value to the request.
+     *
+     * @param array $args
+     * @param string $value
+     * @return array
+     */
+    private function addResourcePrefixHeader(array $args, $value)
```

Comment:
I remember this being a specific requirement for Spanner/Firestore. If it is something we would also like to have handled for Bigtable we can update this method to be shared somewhere.